stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Config.kt

```diff
@@ -109,4 +116,25 @@ public data class Config(
      * The commands that are available on this channel type, e.g. /giphy.
      */
     val commands: List<Command> = mutableListOf(),
-)
+) {
+    val createdAt: Date?
+        get() = created_at
+
+    val updatedAt: Date?
+        get() = updated_at
+
+    val isTypingEventsEnabled: Boolean
+        get() = isTypingEvents
+
+    val isReadEventsEnabled: Boolean
+        get() = isReadEvents
+
+    val isConnectEventsEnabled: Boolean
+        get() = isConnectEvents
+
+    val isSearchEnabled: Boolean
+        get() = isSearch
+
+    val isMuteEnabled: Boolean
+        get() = isMutes
```

Comment:
The style of these properties doesn't match other existing names in this class, e.g. `urlEnrichmentEnabled` and `customEventsEnabled`. Let's make these consistent.